SPECIFICATION.md

```diff
@@ -60,6 +60,8 @@ Swipe is NOT
 - **Percent**: "NN%" relative to the parent
 - **Path**: SVG-style Path String
 - **URL**: relative or absolute URL
+- **StringId**: Regular string used as an identifier
+- **LangId**: Language identifier, such as "*", "en", "ja", "de", etc., where "*" represents the default
```

Comment:
Thanks.  Got it in next commit

Pete

> On May 3, 2016, at 6:18 PM, Yoichi Tagaya <notifications@github.com> wrote:
> 
> In SPECIFICATION.md:
> 
> > @@ -60,6 +60,8 @@ Swipe is NOT
> >  - **Percent**: "NN%" relative to the parent
> >  - **Path**: SVG-style Path String
> >  - **URL**: relative or absolute URL
> > +- **StringId**: Regular string used as an identifier
> > +- **LangId**: Language identifier, such as "*", "en", "ja", "de", etc., where "*" represents the default
> An asterisk needs escaping with a backslash in markdown.
> 
> "\*", "en", "ja", "de", etc., where "\*"
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>